src/relation_engine_server/main.py

```diff
@@ -76,11 +76,11 @@ def view_does_not_exist(err):
 def validation_error(err):
     """Json Schema validation error."""
     resp = {
```

Comment:
Is the error structure returned from the re api documented somewhere? Looking at the test, it was hard for me to figure out what the fields mean.